src/lib.rs

```diff
@@ -95,6 +118,7 @@ pub use request::SentRequest;
 pub use request_tracker::RequestTracker;
 pub use server::Server;
 pub use server::ServerListener;
+pub use transport::Connect;
```

Comment:
In the case of `tranport::Connect`, it must be used directly because it is a trait. If it's not in scope you can't use items from the trait.

For types and functions, it's because I don't want to write the full path everywhere :p

Reply:
Just for my understanding, this could be an option as well right. 

transport::{Connect, ...}

Reply:
Oh, yeah, that would be possible too. I had a little disagreement with `rustfmt` why I didn't do that, but there is a different fix for that too.